### PR TITLE
More reliable `AllowedResourceAliasCheckerTest`

### PR DESCRIPTION
**Pull request by joakime**
Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

### CODE REVIEW

File: tests/test-integration/src/test/java/org/eclipse/jetty/test/AllowedResourceAliasCheckerTest.java [hunk 1]
```diff
@@ -29,42 +27,35 @@
...
     public void start() throws Exception
     {
         _server.start();
         _client.start();
     }
 
-    @BeforeAll
-    public static void beforeAll() throws Exception
+    @BeforeEach
+    public void beforeAll(WorkDir workDir)
```
Comment (sbordet) [inline tests/test-integration/src/test/java/org/eclipse/jetty/test/AllowedResourceAliasCheckerTest.java]: Rename to `before()` or `prepare()`.

File: tests/test-integration/src/test/java/org/eclipse/jetty/test/AllowedResourceAliasCheckerTest.java [hunk 2]
```diff
@@ -76,46 +67,39 @@ public static void beforeAll() throws Exception
...
-    {
-        _client.stop();
-        _server.stop();
+        _baseDir = workDir.getEmptyPathDir().resolve("baseDir");
+        _context.setBaseResource(new PathResource(_baseDir));
     }
 
     @AfterEach
-    public void afterEach()
+    public void afterAll() throws Exception
```
Comment (sbordet) [inline tests/test-integration/src/test/java/org/eclipse/jetty/test/AllowedResourceAliasCheckerTest.java]: Rename to `after()` or `dispose()`.

**Review by sbordet — CHANGES_REQUESTED** [review]

File: tests/test-integration/src/test/java/org/eclipse/jetty/test/AllowedResourceAliasCheckerTest.java (hunk 1 above)
Comment (joakime) [reply tests/test-integration/src/test/java/org/eclipse/jetty/test/AllowedResourceAliasCheckerTest.java]: This one is currently called `public void startServer(WorkDir workDir)`

File: tests/test-integration/src/test/java/org/eclipse/jetty/test/AllowedResourceAliasCheckerTest.java (hunk 2 above)
Comment (joakime) [reply tests/test-integration/src/test/java/org/eclipse/jetty/test/AllowedResourceAliasCheckerTest.java]: this one is currently called `public void teardown()`

**Review by sbordet — DISMISSED** [review]

File: tests/test-integration/src/test/java/org/eclipse/jetty/test/AllowedResourceAliasCheckerTest.java (hunk 1 above)
Comment (joakime) [reply tests/test-integration/src/test/java/org/eclipse/jetty/test/AllowedResourceAliasCheckerTest.java]: now called `public void setupContext(WorkDir workDir)`

File: tests/test-integration/src/test/java/org/eclipse/jetty/test/AllowedResourceAliasCheckerTest.java (hunk 2 above)
Comment (joakime) [reply tests/test-integration/src/test/java/org/eclipse/jetty/test/AllowedResourceAliasCheckerTest.java]: now called `public void dispose()`

**Review by sbordet — APPROVED** [review]

**Pull request merged**